requirements.txt

```diff
@@ -1,5 +1,6 @@
 # media_types dependencies
 boto3 >= 1.7.13
 crc32c
+moto
```

Comment:
This should be in `requirements-dev.txt`, not `requirements.txt`, no?

Reply:
Done